src/main/java/MediumDificultyAssignment/PrimeNumber.java

```diff
@@ -0,0 +1,27 @@
+package MediumDificultyAssignment;
+import java.util.Arrays;
```

Comment:
need space between package and import line

Reply:
done